spring-cloud-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/SentinelProperties.java

```diff
@@ -69,13 +75,7 @@
 	private Metric metric = new Metric();
 
 	/**
-	 * Web servlet configuration when the application is web, the configuration is
```

Comment:
Don't remove it, refer https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java#L142.

Reply:
If it is not deleted, do we need to support this configuration item `spring.cloud.sentinel.servlet. block-page` ?

Reply:
`spring.cloud.sentinel.servlet.block-page` annotated by `DeprecatedConfigurationProperty` and `spring.cloud.sentinel.block-page` is a new item.